Documentation/Metrics/arangodb_server_statistics_server_uptime_total.yaml

```diff
@@ -2,7 +2,7 @@ name: arangodb_server_statistics_server_uptime_total
 help: |
   Number of seconds elapsed since server start
 unit: s
-type: counter
+type: gauge
```

Comment:
I suggest to leave it a counter, if we make it into a gauge, we **must** rename it since it must no longer be called `..._total` (otherwise promtool will complain).